src/components/edge.js

```diff
@@ -425,7 +425,7 @@ class Edge extends React.Component<IEdgeProps> {
   ) {
     let response = Edge.getDefaultIntersectResponse();
 
-    if (trg[nodeKey] == null) {
+    if (trg == null || trg[nodeKey] == null) {
```

Comment:
Fixes a minor console error that shows up when a node is deleted and the connected edges try to calculate their positions.